src/utils/sortPeople.ts

```diff
@@ -0,0 +1,28 @@
+import { Person } from '../types';
+
+export const sortPeople = (sortParam: string, people: Person[]) => {
+  switch (sortParam) {
```

Comment:
The function 'sortPeople' is currently mutating the original 'people' array. This could lead to unexpected results in other parts of your application. To fix this, you should return a new sorted array instead of modifying the original one.

Reply:
Not fixed 
![image](https://github.com/mate-academy/react_people-table-advanced/assets/99403198/8d1b6cdf-c7e2-47da-9b90-fc4357b77822)